### DISCUSSION TITLE
自動ログイン時に使用している情報について

### DISCUSSION THREAD

**Discussion by meganeroppy** [pixiv/vroid-sdk-developers#64] [Q&A]
自動ログイン処理について質問です。
SDKのサンプルをそのまま使用してログイン処理を実装しているのですが、自動ログインの処理の詳細をお伺いしたいです。
具体的には、手動ログイン時に保存し、自動ログイン時に使用している情報は、どこに、なにが保存されているのか、差し支えなければご教示いただけないでしょうか。
他サービスでは同様の処理にアクセストークンを使用するケースが多くみられるようですが、VRoidSDKにおいてもこれを使用しているのでしょうか。
ご回答いただけますと幸いです。
よろしくお願いいたします。

**Comment by nyamadan** [comment]
回答が遅れてしまい申し訳ございません。

> SDKのサンプルをそのまま使用してログイン処理を実装しているのですが、自動ログインの処理の詳細をお伺いしたいです。
具体的には、手動ログイン時に保存し、自動ログイン時に使用している情報は、どこに、なにが保存されているのか、差し支えなければご教示いただけないでしょうか。
> 他サービスでは同様の処理にアクセストークンを使用するケースが多くみられるようですが、VRoidSDKにおいてもこれを使用しているのでしょうか。

こちら、社内で協議したのですが回答することができないとなりました。
ご理解いただけますと幸いです。

**Reply by meganeroppy** [reply]
お返事ありがとうございます。
ご回答は難しいとのこと、承知いたしました。
